proto/tcommit/manager.proto

```diff
@@ -0,0 +1,28 @@
+syntax = "proto3";
+
+import "tcommit.proto";
+
+package degitx.tcommit;
+
+option go_package = "cqfn.org/degitx/proto/go/tcommitpb";
```

Comment:
If we decide to use different Go packages for proto generated files, then let's use hierarchical structure for related packages, e.g. `cqfn.org/degitx/proto/go/tcommit/pb` or `cqfn.org/degitx/proto/go/tcommit/grpc` for `cqfn.org/degitx/proto/go/tcommit` source package.

Reply:
@g4s8  Do we really need to separate *_grpc.pb.go and *.pb.go files into individual directories?

We have all generated go code inside /proto/go separated by directories named by import paths. All files inside directory have the same go package.  For better understandability I propose to add "pb" at the end of the package name. 

There is no recommended solution. 
But looks like it was not intended by Google to separate generated files.

Could you please explain the goals and benefits of your proposed solution?

Reply:
Go prohibits circular dependencies by design, and usually packages are accessed from top to bottom of the hierarchy. E.g. if we have package `a` and a sub-package `a/b` and if we access `a/b` from `a`, then package `a/b` can't access package `a`. So usually we have some common packages with interfaces (`tcommit`) and implementation of these interfaces (without direct access to upper level package) `tcommit/grpc`, and we use it from top-level package (accessing `tcommit/grpc` from `tcommit`), so this way we deny any access from `tcommit/grpc` package to upper level packages. This way, we enforce one-directional package access from top to bottom to achieve better (lower) component coupling.